.github/workflows/publish-prerelease.yml

```diff
@@ -22,9 +22,10 @@ jobs:
       - name: Get merge base commit hash
         id: get-merge-base
         env:
-          BASE_REF: ${{ github.event.pull_request.base.ref }}
```

Comment:
Switched from `ref` to `sha` so that this is less dependent upon the branch having been fetched in the checkout step.

Currently it is fetch, as the checkout action will fetch all branches when you specify a fetch-depth of 0. But I wanted this to be resilient to that changing in the future.